Change the call + FAQs (#595)
ropensci/roweb3: content/champions/_index.md

{{< quote align="center">}}
**The 2023 calls for Champions and Mentors is open!.** 
{{< /quote >}}

## Tentative Program timeline
{{< figure src = "/images/volunteer/peep8.png" width = "300" alt = "A person with his dog working on a notebook." class = "pull-left" >}}

The table below outlines a draft timeline for year one of the Champions Program. As this is a pilot year, there may be minor changes to this.
Research software *should serve everyone in our communities*, which means it needs to be *sustainable, open, and built by and for all groups*. Currently, however, there is a dismaying lack of diversity in open source communities in general and the R / research software community in particular. Developers in the R community are overwhelmingly white, male, and from a handful of countries. That disappointing lack of diversity is potentially detrimental to the sustainability, utility and productivity of projects.

> |Date|Event|Detail|Duration|

Reviewer: I would try to have this schedule in place for the 2023 cohort before announcing the call. It can be really useful for people who need to plan ahead

Author: Yes, I agree it is useful, but we don't need it to open the call. For the call, we need the schedule of the call, but not the full year of the program.  That one I have it a month level, to reach day level I need to agree with several more people.

Reviewer: I think the month level would definitely be sufficient!

> You need to fill out an application form. [This template shows the information we will require](/champions/files/champions_template). The application forms have to be completed *in English*.

Reviewer: ```suggestion
Please fill out an application form! [This template shows the information we require](/champions/files/champions_template). Applications need to be submitted *in English*.
```

> You need to fill out an application form. [This template shows the information we will require](/champions/files/champions_template). The application forms have to be completed *in English*.

Reviewer: Need a link to the form

### Mentors

> You need to fill out an application form. [This template shows the information we will require](/champions/files/mentors_champions_template)). The application forms have to be completed *in English*.

Reviewer: ```suggestion
Please fill out an application form! [This template shows the information we will require](/champions/files/mentors_champions_template). Applications need to be submitted *in English*.
```

> You need to fill out an application form. [This template shows the information we will require](/champions/files/mentors_champions_template)). The application forms have to be completed *in English*.

Reviewer: Need a link to the form

> ## Tentative Program timeline

Reviewer: Will the program_timeline.md file be added here?

## Do you have any questions?

Please, contact our Community Manager [by email](mailto:yabellini@ropensci.org).

> You can also check our resources about the program in English and Spanish.

Reviewer: At the end of this page in the preview are listed all the 2022/2023 champions and mentors... Because there will be overlapping cohorts this year, perhaps @maelle needs to set up a slightly different system for displaying the photos?

### Champions

If you want to be one of our champions [here you can find goals, elegibility, benefits, and expectations for our program](programdetails_2023)

Please, fill out the [application form](https://airtable.com/shrAsYlSXU0coJ5Ld) before __Monday, Sep 4, 2023__.

The application forms have to be filled in _English_. This [template](/champions/files/champions_template) shows the information we require.

### Mentors

Mentors support our cohort of rOpenSci Champions. Could you offer insight and advice to our selected Champions? [Learn the details and express your interest](programdetailsmentors_2023) by filling out the [application form](https://airtable.com/shrlrLdQbb4wphosK) before __Monday, Sep 4, 2023__.

The application forms have to be filled in _English_. This [template](/champions/files/mentors_champions_template) shows the information we require. 

## Application 2023 and Program 2023-2024 timeline

|Date|Event|
|----|-----|
|Jul 14, 2023|Call for Champions and Mentor Applications opens|
|Jul 25, 2023| [OPTIONAL] [Community Call](/commcalls/july2023-championprogram/) - Learn from other mentors and champions and get tips for submitting your application |

> |Aug 1, 2022| [OPTIONAL] [Co-working hours](/events/coworking-2023-08/): receive help with your application to the Champions Program |

Author: ```suggestion
|Aug 17 & 23| [OPTIONAL] Coworking Champions Application Clinics in different timezones ([Aug 17](/events/coworking-2023-08-17/) and [Aug 23](/events/coworking-2023-08-23/)): receive help with your application to the Champions Program |
```
